package/samplers/moead/README.md

```diff
@@ -0,0 +1,74 @@
+---
+author: Hiroaki Natsume
+title: MOEA/D sampler
+description: Sampler using MOEA/D algorithm. MOEA/D stands for "Multi-Objective Evolutionary Algorithm based on Decomposition.
+tags: [sampler, multiobjective]
+optuna_versions: [4.0.0]
+license: MIT License
+---
+
+## Abstract
+
+Sampler using MOEA/D algorithm. MOEA/D stands for "Multi-Objective Evolutionary Algorithm based on Decomposition.
+
+This sampler is specialized for multiobjective optimization. The objective function is internally decomposed into multiple single-objective subproblems to perform optimization.
+
+## Class or Function Names
+
+- MOEADSampler
+
+## Installation
+
+```
+pip install scipy
```

Comment:
This is a nit, but it might be kind to users if we also provided a common way to install the package's dependencies.
 
```
pip install -r https://hub.optuna.org/samplers/moead/requirements.txt
```